docs/cpan-starjacking.md

```diff
@@ -0,0 +1,149 @@
+# StarJacking
+## Rationale
+StarJacking is an undertaking to make people prefer a module because of manipulated popularity. See [StarJacking](https://checkmarx.com/blog/starjacking-making-your-new-open-source-package-popular-in-a-snap/)
+
+While a known topic, checking CPAN for "StarJacking" was mainly the result of the attention drawn to it thanks to experiments from Eugene Rojavski ([EUGENER](https://metacpan.org/author/EUGENER))
+
+## How to choose what CPAN module to depend on?
+CPAN is vast like an ocean, it's sometimes difficult to make a decision about the best module to use. 
+
+Some lists are available like [Task::Kensho](https://metacpan.org/pod/Task::Kensho), [Awesome Perl](https://github.com/hachiojipm/awesome-perl) or [Awesome Perl](https://github.com/uhub/awesome-perl) (again) but they definitely don't cover all needs.
+
+Naming of modules can also help, for instance `Mojo::*` modules should be, by convention, mainly using other `Mojo` inner circle modules as dependencies. `*::Tiny` are generally smaller alternative to other modules of the same purpose, etc...
+
+Most of the time and when possible, user will choose modules and authors that have an established "reputation". This reputation gives them authority on a topic, and the trust of users in term of security.
+
+On top of that, some indicators of quality (build reports and known problems) and activity (maintainance) can be taken into account.
+
+## Popularity indicators in CPAN
+Beyond reputation (module and/or author), MetaCPAN provides some popularity indicators for CPAN modules:
+- ++
+![plusplus](../media/starjacking/plusplus.png)
+
+- [river](https://neilb.org/2015/04/20/river-of-cpan.html) position
+![river](../media/starjacking/river.png)
+
+## Quality indicators in CPAN
+Some other indicators can help in the choice but give more informations about *quality*:
+- Issues
+![Issues](../media/starjacking/issues.png)
+- CPANTesters
+![CPAN Testers](../media/starjacking/testers.png)
+- Bus factor
+![Bus factor](../media/starjacking/bus.png)
+
+Interpretation of indicators "issues" and "bus factor" is debatable.
+
+## Types of attacks
+Link popular repository:
+- "StarJacking" - Attach very popular github repository to module
+
+Craft fake popularity:
+- "PlusPlusJacking" - Manipulate ++ indicator
+- "RiverJacking" - Manipulate river score
+
+## StarJacking ![Stars](../media/starjacking/stars.png)
+StarJacking refers to the trust that VCS stars provides to a module. 
+
+Reusing the popularity of another project, unrelated to the module, 
+and exploiting the non-verification of the link (metadatas) in order make a module falsely popular.
+
+Other ecosystems like PyPI and NPM are also impacted by this problem.
+
+Like other ecosystems, we can attach "metadatas" (repository, website, issue tracker...) to CPAN packages.
```

Comment:
I'd rephrase this as:

Like many other programming language ecosystems, we can attach "metadata" such as a public repository, website and issue tracker URLs to CPAN packages.